uisdk/src/main/java/com/karhoo/uisdk/screen/booking/booking/bookingrequest/BookingRequestActivity.kt

```diff
@@ -161,47 +155,24 @@ class BookingRequestView @JvmOverloads constructor(context: Context,
     override fun showBookingRequest(quote: Quote, outboundTripId: String?, bookingMetadata:
     HashMap<String, String>?) {
         bookingRequestButton.onLoadingComplete()
-        visibility = View.VISIBLE
+        bookingRequestLayout.visibility = View.VISIBLE
```

Comment:
after we import the View we can just say `bookingRequestLayout.visibility = VISIBLE`